packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.vue

```diff
@@ -23,19 +28,24 @@ import GridOverlay from "./GridOverlay/GridOverlay.vue";
 
 const viewportSize = useResizeObserver(shallowRef(document.body));
 
+type MaxWidth = OnyxBreakpoint | "Filled";
+type MaxColumns = 4 | 8 | 12 | 16 | 20;
```

Comment:
`4`and `8` don't exists as options for the maximum column count
```suggestion
type MaxColumns = 4 | 8 | 12 | 16 | 20;
```